Cargo.lock

```diff
@@ -314,12 +323,12 @@ dependencies = [
 
 [[package]]
 name = "flate2"
-version = "1.0.28"
+version = "1.0.33"
```

Comment:
Consider downgrading to 1.0.31, so the dep miniz_oxide is not duplicated

Reply:
Okay done